### ISSUE TITLE
在vue中使用需要哪些插件

### ISSUE THREAD

**Issue by gostfather** [antvis/g6#167]
我在vue中使用统一分层布局 DAGRE 应该装什么

**Comment by TomHuangCN** [comment]
对 vue 不是很了解。但 G6 是基于原生 JS 写的，应该是能在 vue 里跑。https://antv.alipay.com/zh-cn/g6/1.x/demo/other/plugin-dagre.html
